tests/scheduler/scheduler-edit-dialog.html

```diff
@@ -118,72 +104,6 @@
                                     </div>
                                 </div>
                             </div>
-                            <div class="k-recur-view">
-                                <div class="k-scheduler-recurrence-interval-editor">
-                                    <div class="k-form-field">
-                                        <label class="k-label k-form-label">Repeat every</label>
-                                        <div class="k-form-field-wrap">
-                                            <span class="k-recur-interval k-numerictextbox k-input k-input-md k-input-solid k-rounded-md">
-                                                <input type="text" class="k-input-inner" placeholder="" autocomplete="off" value="">
-                                                <span class="k-spin-button k-input-spinner">
-                                                    <button class="k-spinner-increase k-button k-button-md k-button-solid k-button-solid-base k-icon-button">
-                                                        <span class="k-button-icon k-icon k-i-arrow-n"></span>
-                                                    </button>
-                                                    <button class="k-spinner-decrease k-button k-button-md k-button-solid k-button-solid-base k-icon-button">
-                                                        <span class="k-button-icon k-icon k-i-arrow-s"></span>
-                                                    </button>
-                                                </span>
-                                            </span>
-                                            <span>day(s)</span>
-                                        </div>
-                                    </div>
-                                </div>
```

Comment:
we need to have the reference for the recurrence editors as well, please bring them back